PNA/agilentN5242A.py

```diff
@@ -272,7 +274,8 @@ def freq_sweep(self, c, log=False):
         if sweeptime > 1:
             sweeptime *= self.sweepFactor(c)
             print "sweeptime = ", sweeptime
-            yield util.wakeupCall(sweeptime)  #needs factor of 2 since it runs both forward and backward 
+            # needs factor of 2 since it runs both forward and backward
```

Comment:
What factor of two is this?  I know you are just reformatting, but this comment makes no sense and should be removed or fixed.